src/NetAnalyzers/Core/Microsoft.NetCore.Analyzers/Performance/UseCountProperly.cs

```diff
@@ -694,17 +697,19 @@ private static bool TryGetZeroOrOneConstant(IOperation operation, out int consta
             return constant is 0 or 1;
         }
 
-        private static bool IsPropertyGetOfIsEmptyUsingThisInstance(OperationAnalysisContext context, IOperation operation, ISymbol isEmptyPropertySymbol)
+        /// <summary>
+        /// This ensures that we don't suggest any circularity.
+        /// </summary>
+        private static bool IsPropertyGetterUsingThisInstance(OperationAnalysisContext context, IOperation operation, ISymbol propertySymbol)
         {
             ISymbol containingSymbol = context.ContainingSymbol;
 
-            return containingSymbol is IMethodSymbol methodSymbol &&
-                // Is within the body of a property getter?
-                methodSymbol.MethodKind == MethodKind.PropertyGet &&
-                // Is the getter of the IsEmpty property.
-                methodSymbol.AssociatedSymbol == isEmptyPropertySymbol &&
+            // Is within the body of a property getter?
+            return containingSymbol is IMethodSymbol { MethodKind: MethodKind.PropertyGet } methodSymbol &&
+                // Is the getter of the given property.
+                SymbolEqualityComparer.Default.Equals(methodSymbol.AssociatedSymbol, propertySymbol) &&
                 // Is 'this' instance?
-                operation.GetInstanceType() == containingSymbol.ContainingType;
+                SymbolEqualityComparer.Default.Equals(operation.GetInstanceType(), containingSymbol.ContainingType);
```

Comment:
I assume this should have produced an RSxxxx warning if the warning was enabled. (Currently disabled due to a codefix crash)

Reply:
If the problem is only the code-fix, I don't see why we can't enable it on the repo. There are quite some places where I think issues would be raised.

Reply:
@Evangelink There are **lots**. I tend to agree it shouldn't have been disabled in the first place, but given the current number of violations, we need the fixer to fix them and re-enable the rule.

Reply:
We could enable the issues as `Message` so that we can explicitly check for them without breaking the build. Also looks to me that fixing the code-fix should have a high(er) priority.